content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -49,7 +49,7 @@ Cluster administrator actions include:
 
 - [Draining a node](/docs/tasks/administer-cluster/safely-drain-node/) for repair or upgrade.
 - Draining a node from a cluster to scale the cluster down (learn about
-[Cluster Autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaler)
+[Cluster Autoscaling](/docs/tasks/run-application/horizontal-pod-autoscale/)
```

Comment:
This is the wrong link. We need to link to a page that is about automatically managing the number of nodes in the cluster.

I would hyperlink to https://github.com/kubernetes/autoscaler/#readme at least for now.

Reply:
Made the changes

Reply:
I have made the changes